.github/workflows/cli-build-instructions.yml

```diff
@@ -118,6 +118,14 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Upload UnicodeTestResults artifact
+        if: always()
+        uses: actions/upload-artifact@v3
```

Comment:
Observation: This looks good. The artifact will be available for manual download when viewing the page for this particular workflows run instance.

If we want to do something fancier (have that HTML be always visible and up-to-date at a particular URL), we could look at that in a later issue.

Reply:
> have that HTML be always visible and up-to-date at a particular URL

Oooh, yeah I like that idea, like publish it via Pages or something? Maybe other reports too... 🔮 future project maybe 😸

Reply:
The artifact for the most recent run on this PR can be found here: https://github.com/unicode-org/unicodetools/actions/runs/2914569939  ("Artifacts" section at the bottom, click the name which will download a .zip containing the HTML file).
I'm open to adding other files from `Generated` if they would be useful. I mainly added this because there was previously no way to see the output of that test from GitHub CI runs and it's handy for troubleshooting Invariant test failures.

Reply:
I suggest adding a sentence to https://github.com/unicode-org/unicodetools/blob/main/docs/build.md#invariant-checking about the CI providing this artifact, and how to fetch it.

Reply:
Done in https://github.com/unicode-org/unicodetools/pull/318/commits/e0e6afa39664db148c02920e66285577a1e1059e